images/win/scripts/Installers/Install-GitHub-CLI.ps1

```diff
@@ -0,0 +1,10 @@
+################################################################################
+##  File:  Install-GitHub-CLI.ps1
+##  Desc:  Install GitHub CLI
+################################################################################
+
+
+Choco-Install -PackageName gh
+
+Import-Module -Name ImageHelpers -Force
```

Comment:
I think we should move Import Module to the begin of the file

Reply:
I cloned this code from the others scripts, but i agree it is more convenient to have all imports on top